libp2p/protocol_muxer/multiselect_muxer_interface.py

```diff
@@ -23,6 +23,9 @@ def add_handler(self, protocol: TProtocol, handler: StreamHandlerFn) -> None:
         :param handler: handler function
         """
 
+    def get_protocols(self) -> Tuple[TProtocol, ...]:
```

Comment:
in this case no because we can write the implementation for all such `IMultiselectMuxer` using only things declared in this interface.

it would be different if we didn't commit to `handlers: Dict[TProtocol, StreamHandlerFn]` here